src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Good Job adding the tests for the missing components like mentioned by the previous reviewer , however, one of the newly added tests is failing kindly fix it 🙏 

![image](https://user-images.githubusercontent.com/89282221/203973972-8a1ea81e-6cea-4a89-a0c7-0ede82aa6660.png)

Reply:
Thank you for the feedback.
Issue resolved